packages/xo-web/src/xo-app/home/index.js

```diff
@@ -852,19 +871,34 @@ export default class Home extends Component {
             },
           })
           break
-        case 'JUMP_INTO':
+        }
+        case 'JUMP_INTO': {
           const item = items[this.state.highlighted]
           if (includes(['VM', 'host', 'pool', 'SR'], item && item.type)) {
             this.context.router.push({
               pathname: `${item.type.toLowerCase()}s/${item.id}`,
             })
           }
+        }
       }
     }
   )
 
   // Header --------------------------------------------------------------------
 
+  _setVmsFilter = vmsFilter => {
```

Comment:
We can also store it in the URL.

Reply:
Do you think is better to store it in the URL?

Reply:
To keep the same value on refreshing the page and to be coherent with the other functionalities like the sort.